libr/anal/fcn.c

```diff
@@ -1544,7 +1544,56 @@ R_API int r_anal_fcn_cc(RAnal *anal, RAnalFunction *fcn) {
 	return result;
 }
 
-R_API char *r_anal_fcn_to_string(RAnal *a, RAnalFunction *fs) {
+R_API char *r_anal_function_get_name(RAnalFunction *function) {
+	RAnal *a = function->anal;
+	char *ret = NULL, *arg_i = NULL, *args = strdup ("");
+	// RList *args_list = r_list_newf ((RListFree) free);
+	unsigned int argc = 0, i, j;
+	const char *ret_type = sdb_const_get (a->sdb_types,
+	  r_str_newf ("func.%s.ret", function->name), 0);
+	const char *argc_str = sdb_const_get (a->sdb_types,
+	  r_str_newf ("func.%s.args", function->name), 0);
+
+	if (argc_str) {
+		argc = atoi (argc_str);
+	}
+
+	for (i = 0; i < argc; i++) {
+		arg_i = sdb_get (a->sdb_types,
+		  r_str_newf ("func.%s.arg.%d", function->name, i), 0);
+		// parse commas
+		for (j = 0; j < strlen (arg_i); j++) {
+			if (arg_i[j] == ',') {
+				if (arg_i[j - 1] == '*') {
+					// remove whitespace
+					memmove (&arg_i[j], &arg_i[j + 1],
```

Comment:
same here

Reply:
use arg_i +j + i instead of&  []

Reply:
done but I don't want to disturb travis right now